docs/dev/communication.md

```diff
@@ -0,0 +1,15 @@
+# Communication Guidelines
+
+pyDARN is under the Data Analysis Working Group that has setup some guidelines on [communication](https://superdarn.github.io/dawg/documents/communication-guidelines/) to encourage positive collaborative environment.
+Please follow these guidelines when commenting on GitHub. 
+
+Also please read workflow documentation on issues and [pull requests](dev/PR.md) to communicate properly for collaborative work.
```

Comment:
```suggestion
Also please read workflow documentation on issues and [pull requests](dev/PR.md) to communicate effectively for collaborative work.
```
Just a suggestion on wording, I can image someone saying that they are doing it properly, it's just not the way we do it.